tests/e2e/e2e_test.go

```diff
@@ -116,7 +117,8 @@ func TestIndexer(t *testing.T) {
 	time.Sleep(indexerDelay)
 	err = tests.GetFrom(fmt.Sprintf("/consensus/accounts/%s", bobAddress), &account)
 	require.Nil(t, err)
-	require.Equal(t, uint64(100000000), account.Available)
+	expected := big.NewInt(100000000)
```

Comment:
Can we use number >2^64 in at least one or two places in this test? That will test the DB part of your new code nicely.

Reply:
That would be ideal, but I'd prefer to keep it out of scope for this PR because it requires changes to the test-runner setup and oasis-node, specifically modifying [this value](https://github.com/oasisprotocol/oasis-core/blob/3d338936cd34e72a2662fa91d8e1aeb2ccf8d6d3/go/oasis-node/cmd/debug/txsource/workload/workload.go#L24) in [FundAccountFromTestEntity](https://github.com/oasisprotocol/oasis-core/blob/go/v0.2202.3/go/oasis-node/cmd/debug/txsource/workload/workload.go#L162). Will file an issue to track this.